string  sensorName
float32 indicator
string  sensorType
string  activatorName
string  condition
